serokell/tezos-packaging
# [#134] Move abstract formula to a different dir

## Description

Problem: `brew` tap checks all *.rb files in Formula directory for
being a valid brew Formula. If one of the is invalid, the whole tap is
broken:(
`Formula/tezos.rb` is an abstract class for all other *.rb files and
thus an invalid brew Formula.

Solution: Move tezos.rb do a dedicated directory, thus it won't be
checked during `brew tap`.

To test:
```
brew tap rvem/tezos-packaging https://github.com/rvem/tezos-packaging.git
brew install tezos-client
```
Because `brew tap` always checks a repository default branch, it's hard to check a fix in this repo.
Having a fix in a fork master is an the easiest way to test it.



## Related issue(s)



Resolves #134

#### Related changes (conditional)

- [x] I checked whether I should update the [README](../tree/master/README.md)

- [x] I checked whether native packaging works, i.e. native binary packages
  can be successfully built.

#### Stylistic guide (mandatory)

- [x] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).
